pkg/ring/kv/multi.go

```diff
@@ -43,10 +43,8 @@ type MultiRuntimeConfig struct {
 	// or to gossip). Doing this allows nice migration between stores. Empty values are ignored.
 	PrimaryStore string `yaml:"primary"`
 
-	// Put "enabled", "enable" or "true" here to enable. Any other value = disabled. Empty = ignore.
-	// We don't use bool here, because it would not be possible to distinguish between false = missing value,
-	// or false = disabled.
-	Mirroring string `yaml:"mirroring"`
+	// Mirroring enabled or not. Nil = no change.
+	Mirroring *bool `yaml:"mirroring"`
```

Comment:
Naming wise, shouldn't this reflect `MultiConfig`, so naming it `MirrorEnabled` / ``yaml:"mirror-enabled"`?